src/screening.cpp

```diff
@@ -544,41 +790,43 @@ int main(int nargs, char **argv) {
   mkdir(convert_string(file_storage_location + "processing_files"), 0777);
   mkdir(convert_string(file_storage_location + "processing_files/reservoirs"), 0777);
 
-  if (search_config.search_type.not_existing()) {
-    Model<short> *DEM = read_DEM_with_borders(search_config.grid_square, border);
+  // Create the DEM and filter model
+  Model<bool> *filter;
+  Model<short> *DEM = read_DEM_with_borders(search_config.grid_square, border);
```

Comment:
We don't want to create these unless we need to - the bluefield and single-pit search does not need it

Reply:
Updated to only create these for bulk_pit and greenfield